cypress/integration/common/steps.js

```diff
@@ -551,3 +576,11 @@ Then("I see only one post with the title {string}", title => {
     .should("have.length", 1);
   cy.get(".main-container").contains(".post-link", title);
 });
+
+Then("they should not see the comment from", () => {
+  cy.get(".ds-card-footer").children().should('not.have.class', 'comment-form')
+})
+
+Then("they should see a text explaining commenting is not possible", () => {
+  cy.get('.ds-placeholder').should('contain', "Commenting is not possible at this time on this post.")
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-23T17:08:36Z" title="Thursday, January 23rd 2020, 6:08:36 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -550,3 +570,11 @@ Then("I see only one post with the title {string}", title => {
     .should("have.length", 1);
   cy.get(".main-container").contains(".post-link", title);
 });
+
+Then("they should not see the comment from", () => {
+  cy.get(".ds-card-footer").children().should('not.have.class', 'comment-form')
+})
+
+Then("they should see a text explaining commenting is not possible", () => {
+  cy.get('.ds-placeholder').should('contain', "Commenting is not possible at this time on this post.")
```


That's quite important to avoid false negatives :+1: I told my students yesterday that they should look for something existing in the test before looking for something that does not exist, like in the step definition above :point_up: